cloud-optimized-geotiffs/environment.yml

```diff
@@ -7,5 +7,6 @@ dependencies:
   - ipykernel
   - jupyterlab
   - matplotlib
-  - rasterio
-  - rio-cogeo
+  - rasterio>=1.3.3
+  - rio-cogeo>3.5
+  - gdal>=3.5
```

Comment:
Do we need `gdal` or just `libgdal`? The former also includes the built-in python package, I think?

Reply:
This is probably a remnant of an old version where I used `gdalinfo`. So maybe swap (though would need to test again), however we might want to leave it if we plan to include any shell examples in the COG section, which I'm pretty sure we will #91

Reply:
ok test with `libgdal` and works fine. Also fixed some text at the same time.